src/client/app/containers/editor.ts

```diff
@@ -39,6 +39,17 @@ export class Editor implements AfterViewInit {
      */
     async ngAfterViewInit() {
         this._monacoEditor = await this._monaco.create(this._editor, { theme: 'vs' });
+        let editor = this._monacoEditor;
+        editor.addAction({
+            id: 'trigger-suggest',
+            label: Strings.triggerActionLabel,
+            keybindings: [monaco.KeyCode.F2],
+            keybindingContext: null,
+            contextMenuGroupId: 'navigation',
+            // Put action at top of context menu
```

Comment:
Could you make it be a `/* ... */` sort of comment?  I.e.,:

        contextMenuOrder: 0 /* put at top of context menu */,